### PR TITLE
add file flag to send-event to specify file containing multiple events

### PR DESCRIPTION
**Pull request by NikhilSharmaWe**
### Description
This PR adds  `-f` flag to the `send-event` command for specifying the file containing events.

### Motivation
Fixes: #220

### CODE REVIEW

File: cmd/sendevent/sendevent.go [hunk 1]
```diff
@@ -119,3 +138,24 @@ func (o *CliOptions) send(eventType, target, data string) error {
 	fmt.Printf("\nResponse: %s\n", response)
 	return nil
 }
+
+func readEventsFromFile(file string) ([]string, error) {
+	var rawEvents []json.RawMessage
+
+	fileData, err := ioutil.ReadFile(file)
```
Comment (tzununbekov) [inline cmd/sendevent/sendevent.go:145]: [`ioutil.ReadFile`](https://cs.opensource.google/go/go/+/refs/tags/go1.20.5:src/io/ioutil/ioutil.go;l=35) is deprecated, please switch to `os.ReadFile`:
```suggestion
	fileData, err := os.ReadFile(file)
```

**Review by tzununbekov — CHANGES_REQUESTED** [review]
Looks good, only one tiny comment before the merge.

File: cmd/sendevent/sendevent.go (hunk 1 above)
Comment (NikhilSharmaWe) [reply cmd/sendevent/sendevent.go]: Resolved

**Comment by NikhilSharmaWe** [comment]
@tzununbekov 
Updated the docs

**Review by tzununbekov — APPROVED** [review]

**Pull request merged**